tests/test_bkr_handlers_mocked.py

```diff
@@ -0,0 +1,456 @@
+from __future__ import print_function
+import unittest
+from unittest.mock import patch
+
+import sys
+import time
+import os
+import tempfile
+import shutil
+
+import teres
+import teres.bkr_handlers
+
+my_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
These 2 lines don't do anything and need to be either moved or should be removed.
If they were to work, they would have to be present before the first import of teres (to make the teres available for importing). The tests currently work as the `tests.test_bkr_handlers.BkrTest` is imported first and it contains this sys.path manipulation so teres is available thanks to that, but in such case this insert just adds duplicit path there. It this module was imported as the first one, it would crash on `teres` not being available and wouldn't reach those lines.

The proper way to approach the tests is to run them from the project root directory and in such case, none of these sys.path manipulations is needed. (not subject of this pull request)

Reply:
I admit the `my_path` is a cargo-cult copy-paste. I don't even know why it was needed.

I agree that tests running from project root, and project root only, is a good assumption (if not best practice).

After removing explicit import of `teres.bkr_handlers` and the `my_path` manipulation following still works (tested on Fedora 35, rhel-8 and rhel-9, no Beaker):

    python3 -m unittest -bv tests/test_bkr_handlers_mocked.py

so it's probably some workaround related to old unittest...(?)  Anyway, thanks -- I've dropped it.